pkg/gadgets/trace/exec/types/types.go

```diff
@@ -24,12 +24,12 @@ import (
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32   `json:"pid,omitempty" column:"pid,width:7,fixed"`
-	Ppid      uint32   `json:"ppid,omitempty" column:"ppid,width:7,fixed"`
+	Pid       uint32   `json:"pid,omitempty" column:"pid,minWidth:7"`
+	Ppid      uint32   `json:"ppid,omitempty" column:"ppid,minWidth:7"`
 	Comm      string   `json:"pcomm,omitempty" column:"comm,width:16,fixed"`
```

Comment:
As discussed [here](https://github.com/kinvolk/inspektor-gadget/pull/934#discussion_r977458270), I think COMM should be: `column:"comm,maxWidth:16"`. We can decide later whether we should add a `minWidth` or not.